### ISSUE TITLE
Output to stdout?

### ISSUE THREAD

**Issue by demian85** [wulkano/aperture.js#18]
I hope you are not hating me right now for asking for many things, but I'm having serious issues with my project and this module is still my main option to move forward.

By any chance, is there an easy way to output video chunks to stdout so I can easily pipe to ffmpeg and fix moov atom (progressive download or faststart, whatever it's called)?

I'd appreciate if you can give me a hint.

Thanks!

**Comment by matheuss** [comment]
@demian85 I think you could take the `filePath` that we pass when resolving the promise for `startRecording` and then read the file while it grows 🤔

**Comment by demian85** [comment]
yeah, kinda hackish, and I cannot even find out how to read from stdin with ffmpeg, you need to be Einstein.
